packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -433,6 +433,44 @@ export const VtexCommerce = (
           { storeCookies }
         )
       },
+      delete: async (reviewId: string): Promise<true> => {
+        const review = await fetchAPI(
+          `${base}/${REVIEWS_AND_RATINGS_API_PATH}/review/${reviewId}`,
+          undefined,
+          { storeCookies }
+        )
+
+        if (!review) {
+          throw new NotFoundError(`Review not found for id ${reviewId}`)
+        }
+
+        const authCookieKey: string = `VtexIdclientAutCookie_${account}`
+
+        const authCookie = getCookieFromRequestHeaders(ctx, authCookieKey) ?? ''
+
+        if (!authCookie) {
+          throw new UnauthorizedError('Missing auth cookie')
+        }
+
+        const result = await fetchAPI(
+          `${base}/${REVIEWS_AND_RATINGS_API_PATH}/review/${reviewId}`,
+          {
+            ...BASE_INIT,
+            headers: {
+              ...BASE_INIT.headers,
+              VtexIdclientAutCookie: authCookie,
+            },
+            method: 'DELETE',
+          },
+          { storeCookies }
+        )
+
+        if (result.toLowerCase() === 'invalid user') {
```

Comment:
I've got error because when result is fine it returns a boolean. I believe this snippet was already here and got lost in some commit:
```suggestion
        if (typeof result === 'string' && result.toLowerCase() === 'invalid user') {
```